src/main/scala/nl/knaw/dans/api/sword2/DepositProperties.scala

```diff
@@ -65,8 +65,8 @@ object DepositProperties {
   private def readProperties(f: File) = {
     val ps = new PropertiesConfiguration()
     ps.setDelimiterParsingDisabled(true)
-    ps.load(f)
-
+    if(f.exists) ps.load(f)
```

Comment:
This methods sets the current deposit state, so if `deposit.properties` doesn't exist it must be created.